fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/typing/Type.java

```diff
@@ -281,7 +283,7 @@ private static Type fromProtoType(@Nullable Descriptors.GenericDescriptor descri
             // collection type
             // case 1: primitive types -- assumed to be non-nullable array of that type
             if (typeCode.isPrimitive()) {
-                final var primitiveType = primitiveType(typeCode, false);
+                final var primitiveType = primitiveType(typeCode, true);
```

Comment:
I set all `isNullable` to `true` in this PR, because I think that's what we support today? We haven't supported Nonnull yet right?